packages/storybook/.storybook/main.js

```diff
@@ -3,10 +3,25 @@ const path = require('path');
 module.exports = {
   stories: ['../@(src|stories)/**/*.stories.@(js|jsx|ts|tsx|mdx)'],
   addons: [
+    {
+      name: '@storybook/addon-docs',
+      options: {
+        sourceLoaderOptions: {
+          injectStoryParameters: false,
```

Comment:
This allows the storysource addon to display the entire story in the Canvas Story tab.